intermediate_html_css/grid/advanced_grid_properties.md

```diff
@@ -279,7 +279,7 @@ Notice how when we resize, the columns automagically know how many will fit acro
 
 So what's going on here specifically with `repeat(auto-fit, minmax(150px, 1fr));`? Simple! Remember that `auto-fit` will return the **highest positive integer** without overflowing the grid. So first, the browser has to know how wide our grid is: in this case, it's just the window's width (minus margins) because we didn't explicitly set it. For the sake of this example, let's pretend like our window is currently `500px` wide. Second, the browser needs to know how many grid column tracks could possibly fit in that width. To do this, it uses the minimum value in our `minmax()` function, since that will yield the highest number of items, which is `150px`. If our window is `500px` wide, this means our grid will render 3 columns. But wait, there's more! Once the browser has determined how many columns we can fit, it then resizes our columns up to the maximum value allowed by our `minmax()` function. In this case, our max size is `1fr`, so all three columns will be given an equal allotment of the space available. As we resize our window, these calculations happen in realtime and the result is what you see in the above example!
 
-#### What about `auto-fill`?
+#### What about auto-fill?
```

Comment:
thanks for spotting and correcting these! I've noticed a few more code blocks in the headers here. I think those are worthy of their own issue as well since this happens throughout multiple lessons. Not something required by the description of my issue, but indeed headers should not contain code formatting according to the style guide.